subscription-offer-prompt/index.js

```diff
@@ -65,5 +66,23 @@ function createSubscriptionPrompt (html) {
 	};
 }
 
-export const initPrompt = executeWhen(createSubscriptionPrompt(createPopupHTML));
+function getSubscriptionPromptValues () {
+	const currencyCode = 'GBP';
```

Comment:
Don't think it's possible with ease at present. Once we ditch ie9 & 10 js support, and DAM gets deprecated, I would hope that membership provide us with a CORS enabled service for this sort of thing. As a stop gap you could add a json endpoint to product-selector to get this info